src/Core/tests/Benchmarks/Benchmarks/GestureRecognizerBenchmarker.cs

```diff
@@ -0,0 +1,82 @@
+﻿using BenchmarkDotNet.Attributes;
+using Microsoft.Maui.Controls;
+
+namespace Microsoft.Maui.Handlers.Benchmarks;
+
+[MemoryDiagnoser]
+public class GestureRecognizerBenchmarker
+{
+	View[] _views = null;
+	IGestureRecognizer[] _gestureRecognizers = null;
+
+	private const int Iterations = 100;
+
+	[GlobalSetup]
+	public void Setup()
+	{
+		_views = [
+			new Border(), new BoxView(), new CarouselView(), new Grid(), new Entry(), new Picker(), new CollectionView(),
+			new CheckBox(), new DatePicker(), new Stepper(), new Slider(), new ActivityIndicator(), new Frame(),
+			new ContentView(), new ProgressBar(), new SearchBar(), new Switch(), new TimePicker(), new WebView(), new Button(),
+		];
+
+		_gestureRecognizers = [
+			new PointerGestureRecognizer(), new TapGestureRecognizer(), new PanGestureRecognizer(),
+			new SwipeGestureRecognizer(), new DragGestureRecognizer(), new DropGestureRecognizer(),
+		];
+	}
+
+	[Benchmark]
+	public void CreateLotsOfGestureRecognizers()
```

Comment:
Should we also have a benchmark for 1 recognizer?

Reply:
Done